backend/game/irepository/test_irepository_dice.py

```diff
@@ -0,0 +1,91 @@
+import pytest
+
+from game.irepository.irepository_dice import IRepositoryDice
+from game.irepository.irepository_player import IRepositoryPlayer
+from game.dice_handler import DiceHandler
+from game.player.player import Player
+
+
+@pytest.mark.django_db(transaction=True)
+def test_save_dice():
+    repository_dice = IRepositoryDice()
+    repository_player = IRepositoryPlayer()
+    player = Player()
+    player.set_monster_name('Godzilla1')
+    player.set_username('Test_Get')
+    player.set_password('Pass')
+    repository_player.save_player_db(player)
+    dice = DiceHandler()
+    dice.dice_values = ['1', '2', '3', '1', '2', '3']
```

Comment:
The dice values should use the dice enum

Reply:
swapped to enum